docs/tutorial.rst

```diff
@@ -1092,37 +1126,32 @@ We'll count up the number of neutral and deleterious mutations in the three regi
 .. code-block:: python
 
     num_neutral = np.zeros(3, dtype="int")
-    num_del = np.zeros(3, dtype="int")
+    num_deleterious = np.zeros(3, dtype="int")
     for site in ts.sites():
-        j = int(site.position >= gene_interval[0, 0]) + int(
-            site.position >= gene_interval[0, 1]
-        )
-        unique_muts = {}
+        region = np.digitize(site.position, gene_interval.flatten())
         for mut in site.mutations:
-            for ds, md in zip(mut.derived_state.split(","), mut.metadata["mutation_list"]):
-                if ds not in unique_muts:
-                    unique_muts[ds] = md["selection_coeff"]
+            for mut_info in mut.metadata["mutation_list"]:
+                if mut_info["selection_coeff"] == 0:
+                    num_neutral[region] += 1
                 else:
-                    assert unique_muts[ds] == md["selection_coeff"]
-        for s in unique_muts.values():
```

Comment:
note that previously we were only summing over `unique_muts`, so we weren't double-counting; if we got the same answer previously it means we don't have any mutations happening to stack.

Reply:
yup, got it!